backend/sdk/vnscope/classify.py

```
@@ -0,0 +1,354 @@
+                group, poc, vah, val, total_volume, min_peak_distance
+            )
+            shapes.append({"symbol": symbol[0], "shape": shape})
+
+        # Convert results to DataFrame
+        shapes_df = pl.DataFrame(shapes)
+        return shapes_df
+
+    def plot_heatmap_with_candlestick(self, symbol, number_of_levels, overlap_days):
+        from time import time as now
+        from datetime import datetime, timedelta
+        from matplotlib.colors import LinearSegmentedColormap
```

Ruff (0.11.9)

`matplotlib.colors.LinearSegmentedColormap` imported but unused

Remove unused import: `matplotlib.colors.LinearSegmentedColormap`

(F401)